stream/streams.go

```diff
@@ -795,11 +791,8 @@ func (self *rollingMinByStream) Next() (Event, error) {
 			return nil, EOI
 		}
 
-		if err1 != nil {
-			return nil, err1
-		}
-		if err2 != nil {
-			return nil, err2
+		if err := getError(err1, err2); err != nil {
```

Comment:
If one error is EOI and another isn't, we return list of two, but we want to return EOI and log the other one.